content/klc/F6.3.adoc

```diff
@@ -14,7 +14,7 @@ Surface mount pads have specific requirements for PCB design.
 .. `B.Paste` - Back solderpaste (stencil openings)
 . Pads with specific stencil aperture design requirements require special attention (see details below)
 
-## Requirements for solderpaste
+*Requirements for solderpaste*
```

Comment:
AFAICS this no longer uses a header `<hx>` tag here, which could hurt the semantic structure of the document? Looking at the rendered KLC on the website, it seems that the section headings (i.e. the "brief" in this file) are rendered as h4, so perhaps these should be made h5 instead (i.e. 5 hash signs, I think). Alternatively, the rendering engine could (perhaps) be modified to offset the heading types (e.g. switch to a single `#` and render than as `h5`).

Also, I think this over-sized heading problem affects more than just this section, so it might need to be fixed more generally?

Reply:
I just happened to see this big heading issue here, and it was better in F6.2, so I copied the heading from that guideline.

Reply:
Maybe better to fix all of them separately in a single commit (might be better to do in a followup PR?).

Reply:
Agreed. I'm ready to finish off this PR.